files/sidebars/jssidebar.yaml

```
@@ -377,7 +377,13 @@ l10n:
     Guide_Internationalization: 国际化
     Guide_Meta: 元编程
     Guide_Modules: JavaScript 模块
+    Beginners: 初学者教程
+    Adding_interactivity: 你的第一个网站：添加交互
+    Dynamic_scripting_with_JavaScript: 使用 JavaScript 动态编码
```

```suggestion
    Dynamic_scripting_with_JavaScript: 使用 JavaScript 添加脚本交互
```

(I think it's better to focus on the meaning rather than the literal translation here; my suggestion. Of course this means you need to update the learning sidebar too)